### PR TITLE
SEO fixes of broken links SLE15SP0 EN

### PR DESCRIPTION
**Pull request by dariavladykina**
Applied broken link fixes for SEO purposes.

Changes will be done for each version separately, so no backports needed.

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [ ] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
  - [x] SLE 15 SP0
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4
  - [ ] SLE 12 SP3

### CODE REVIEW

**Comment by jfaltenbacher** [comment]
LGTM, but the conflict needs to be solved.

**Review by jfaltenbacher — COMMENTED** [review]
please solve the conflict first

File: xml/apps_gimp.xml [hunk 1]
```diff
@@ -1113,7 +1113,7 @@
    <listitem>
     <para>
      There are several mailing lists about GIMP. You will find them at
-     <link xlink:href="https://www.gimp.org/mail_lists.html"/>. The GIMP User
+     <link xlink:href="https://www.gimp.org/discuss.html#mailing-lists/"/>. The GIMP User
```
Comment (dariavladykina) [inline xml/apps_gimp.xml:1116]:
```suggestion
     <link xlink:href="https://www.gimp.org/discuss.html#mailing-lists"/>. The GIMP User
```

**Review by dariavladykina — COMMENTED** [review]
Update link

File: xml/apps_gimp.xml [hunk 2]
```diff
@@ -1113,7 +1113,7 @@
    <listitem>
     <para>
      There are several mailing lists about GIMP. You will find them at
-     <link xlink:href="https://www.gimp.org/mail_lists.html"/>. The GIMP User
+     <link xlink:href="https://www.gimp.org/discuss.html#mailing-lists"/>. The GIMP User
```
Comment (dariavladykina) [inline xml/apps_gimp.xml:1116]:
```suggestion
     <link xlink:href="https://www.gimp.org/discuss.html"/>. The GIMP User
```

**Review by dariavladykina — COMMENTED** [review]
Update link

**Pull request merged**